cypress/integration/tests/dashboard/release-notes/release-notes-step-defs.ts

```diff
@@ -0,0 +1,42 @@
+import { Given,When,And,Then } from "cypress-cucumber-preprocessor/steps";
+import ReleaseNotes from '../../../pageobjects/release-notes'
+import LoginPage from '../../../pageobjects/login-page'
+
+const loginPage = new LoginPage()
+const releaseNotes = new ReleaseNotes();
+
+Given('I am already logged in to BMS', ()=> {
```

Comment:
You don't need to implement step definitions for this (it's a common step in `navigation.ts`)
`Given('I am already logged in to BMS', ...`